node/src/commands/genesis/mod.rs

```diff
@@ -183,11 +184,14 @@ fn parse_auth_inputs(
 ) -> Result<(AuthScheme, AuthData)> {
     match auth_scheme_input {
         AuthSchemeInput::RpoFalcon512 => {
-            let auth_seed = hex_to_bytes(auth_seed)?;
-            let keypair = KeyPair::from_seed(&auth_seed)?;
+            let auth_seed: [u8; 40] = hex_to_bytes(auth_seed)?;
+            // Get the first 32 bytes, ignore remaining:
+            let auth_seed: [u8; 32] = auth_seed[..32].try_into().expect("Unreachable");
```

Comment:
Instead of doing this, we should probably just shorten the `auth_seed` string `genesis.toml` file and then we can just do `let auth_seed: [u8; 32] = hex_to_bytes(auth_seed)?;`